ts/features/bonus/bpd/screens/details/transaction/TransactionsUnavailable.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { SafeAreaView } from "react-native";
+import image from "../../../../../../../img/wallet/errors/payment-unavailable-icon.png";
+import { IOStyles } from "../../../../../../components/core/variables/IOStyles";
+import { renderInfoRasterImage } from "../../../../../../components/infoScreen/imageRendering";
+import { InfoScreenComponent } from "../../../../../../components/infoScreen/InfoScreenComponent";
+import BaseScreenComponent from "../../../../../../components/screens/BaseScreenComponent";
+import I18n from "../../../../../../i18n";
+
+export type Props = Pick<
+  React.ComponentProps<typeof BaseScreenComponent>,
+  "contextualHelp"
+>;
+
+const loadLocales = () => ({
+  headerTitle: I18n.t("bonus.bpd.details.transaction.goToButton"),
+  title: I18n.t("bonus.bpd.details.transaction.error.title"),
+  body: I18n.t("bonus.bpd.details.transaction.error.body")
+});
+
+/**
+ * This screen informs the user that there are problem in retrieve the transactions list.
```

Comment:
```suggestion
 * This screen informs the user that there are problems retrieving the transactions list.
```

Reply:
Fixed in [156a52a](https://github.com/pagopa/io-app/pull/2755/commits/156a52ae7391c759b440fc09100092134189e1b1)